drafts/25-xxx-specifications.txt

```diff
@@ -197,73 +197,361 @@ to11. The following tokens also appear in the illustrative syntax below.
       | ID                  | Regex [A-Za-z][A-Za-z0-9_]*               |
       | WHOLE_NUMBER        | Regex [0-9]+                              |
       | REAL_NUMBER         | Fortran 2023 R714 real-literal-constant.  |
-      | CHARACTER_STRING    | Fortran 2023 R724 char-literal-constant.  |
+      | CHARACTER_STRING    | RegEx "[^"\n]*"                           |
```

Comment:
Following up from a discussion point in our meeting, there was a question about corner-case inputs like this:

```
#include "stddef.h\
"
```

It appears this *IS* permitted by C23, because line conjoining removes the backslash-newline _before_ tokenization of the character string. I think we are specifying the same behavior, meaning the regexs in this token list (eg. `CHARACTER_STRING`) are applied AFTER line conjoining.

Agreed?

Reply:
I think that is right. Conjoining takes precedence over tokenization. I thing this also handles Patrick's concern about `CHARACTER_STRING` regex below (and then some).